locations/states_counties.py

```diff
@@ -0,0 +1,613 @@
+state_county = {
```

Comment:
I don't think you need this file anymore.

Reply:
Do u think we might need this for future spiders?